acme/src/acme/errors.py

```diff
@@ -107,6 +107,16 @@ def __init__(self, failed_authzrs: List['messages.AuthorizationResource']) -> No
         self.failed_authzrs = failed_authzrs
         super().__init__()
 
+    def __str__(self) -> str:
+        msg = []
+        for authzr in self.failed_authzrs:
+            msg.append(f'Authorization for {authzr.body.identifier} ' \
+                'failed due to one or more failed challenges:')
+            for challenge in authzr.body.challenges:
+                msg.append(f'  Challenge {challenge.chall.typ} failed ' \
+                    f'with error  {str(challenge.error)}')
```

Comment:
```suggestion
                    f'with error {str(challenge.error)}')
```
did you mean to put two spaces here? if so, please ignore this

Reply:
i absolutely did not, and figured the 2 spaces in the test case output was some weird `__str__` artifact lmao